firebase-app/build.gradle.kts

```diff
@@ -44,6 +44,13 @@ android {
     }
 }
 
+val KonanTarget.archVariant: String
+    get() = if (this is KonanTarget.IOS_X64 || this is KonanTarget.IOS_SIMULATOR_ARM64) {
+        "ios-arm64_i386_x86_64-simulator"
```

Comment:
As I understand it, this slice should be correct for x86 and arm simulators

Reply:
🚨 This doesn't build locally - the issue appears to be at the `:firebase-app:cinteropFirebaseCoreIosArm64` task, which I don't quite understand how to get a verbose output from as it runs in another Java process (I'm probably past my level of understanding here 😅).

I've pushed the changes anyway in case there is something obvious I have missed? @Reedyuk

Reply:
@pauldavies83 It compiles in my machine. I've had that error happen to me a number of times because of cinterop tasks not caching correctly. Try Clean Project, it usually fixes it :)

Reply:
🤔 Hmmm, I get the same error after a clean...

`./gradlew clean build --no-build-cache`

I found an issue from reading the CI log anyway, which I've fixed [here](https://github.com/GitLiveApp/firebase-kotlin-sdk/pull/220/commits/f340a86f90ad27ca095a4b807e57faaa9cf6d0a4)